app/api/csv/typeParsers/relationship.ts

```diff
@@ -4,30 +4,43 @@ import { RawEntity } from 'api/csv/entityRow';
 import { ensure } from 'shared/tsUtils';
 import { PropertySchema } from 'shared/types/commonTypes';
 
+interface Query {
+  title: { $in: string[] };
+  template?: string;
+}
+
```

Comment:
i do not think you really need an interface here? can we just use an object to not redefine what exists? if it is really needed you can use FilterQuery from @types/mongoose